packet/loadbalancers.go

```diff
@@ -30,31 +30,33 @@ type loadBalancers struct {
 	project            string
 	configmapnamespace string
 	configmapname      string
+	configmapenable    bool
 	facility           string
 	localASN, peerASN  int
 }
 
-func newLoadBalancers(client *packngo.Client, projectID, facility string, configmap string, localASN, peerASN int) *loadBalancers {
+func newLoadBalancers(client *packngo.Client, projectID, facility, configmap string, configmapenable bool, localASN, peerASN int) *loadBalancers {
```

Comment:
This is a bit of a strange interface. I have the `configmap` as a string, which I use both to give the name of the configmap and whether it is enabled, and then a `configmapenable` which determines if it is enabled (didn't we just do that?).

It would be good if we could pull the complexity (largely driven by the user-facing settings) out of here entirely, and made this interface easier to understand. Maybe something like these parameters:

* `configmap string` - if blank, do not do configmap stuff; if non-blank, it is the namespace+name of the configmap
* `enabled bool` - if false, do not do any loadbalancer stuff; if true, enable it

This seems easier to understand. Converting the various options of the env var into these can happen at the entrypoint of the entire application.

Reply:
That's fine.. I went down a convoluted route as we default blank to `metallb-system:config`. Will this default now live in the manifest?

Reply:
No need. The default lives in `main.go` where it sets defaults on options.

This drags us back into the #117 discussion as to what the right way this should look to users is. But for now, the blank option is rendered in `main.go`, which deals with CLI flags and converts them into something computer-sane.